### DISCUSSION TITLE
btcpay server opencart 3x integration

### DISCUSSION THREAD

**Discussion by atoshk** [btcpayserver/btcpayserver#2522] [General]
hello 
why not have opencart 3x integration? 
many shop and users already used opencart maybe more than prestashop or wordpress
we ware waiting integrationss for opencart 3x asap 
thanks in advance

**Comment by Kukks** [comment]
You are the first to request OpenCart so I doubt there is enough interest

**Comment by atoshk** [comment]
really https://bitcointalk.org/index.php?topic=4686228.0

If they release plugins for Opencart, Zencart, osCommerce and Magento then basically they have the main players. With the majority of ecommerce websites running on Wordpress/Woocommerce they already have taken a huge step forward to offer a massive percentage of ecommerce traders with their already released plugin.

check it please opencart have many users and shops. if have opencart 3x integrations. really help full for many shops and peoples 

    On Thursday, May 13, 2021, 02:28:41 PM GMT+8, Andrew Camilleri ***@***.***> wrote:  
 
 


You are the first to request OpenCart so I doubt there is enough interest

**Comment by atoshk** [comment]
how much money you want to make opencart 3x plugin ?i am ready to pay reasonable money friend for opencart 3x plugin
    On Thursday, May 13, 2021, 03:14:24 PM GMT+8, Atos Hk ***@***.***> wrote:  
 
  really https://bitcointalk.org/index.php?topic=4686228.0

If they release plugins for Opencart, Zencart, osCommerce and Magento then basically they have the main players. With the majority of ecommerce websites running on Wordpress/Woocommerce they already have taken a huge step forward to offer a massive percentage of ecommerce traders with their already released plugin.

check it please opencart have many users and shops. if have opencart 3x integrations. really help full for many shops and peoples 

    On Thursday, May 13, 2021, 02:28:41 PM GMT+8, Andrew Camilleri ***@***.***> wrote:  
 
 


You are the first to request OpenCart so I doubt there is enough interest

**Reply by woutersamaey** [reply]
It’s safe to say that a quality plugin can easily take 4-5 days of development time, excluding testing and writing of documentation or manual. If you’re interested to hire me, I’m willing to look into it.
I’m just about to finish the new Magento 2 module with Greenfield API support, so OpenCart could be next.

**Comment by woutersamaey** [comment]
I guess the interest has passed?

**Comment by demianrihs** [comment]
I am also interest, any chances this will get done? The plugin can be sold on opencart's marketplace

**Comment by Kukks** [comment]
Duplicate of  #870
